.github/workflows/python-app.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: shub
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+        python-version: 3.8
+        python-version: 3.9
```

Comment:
Why not test the same as Travis CI?

Reply:
I will remove 2.7, 3.5, and 3.6 from TravisCI.

Yet using Github Actions is experimental, and it's not working. I'll probably revert it.

Reply:
Are we delaying the next release until 3.6 goes end of life on 23 Dec?

Reply:
Regarding GitHub Actions, it’s the only way forward, we cannot use Travis CI anymore.